pkg/controller/render/render_controller.go

```diff
@@ -416,6 +461,36 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 		return nil
 	}
 
+	// If it's a pool named "layered", it's special, designate it automatically
+	// TODO(jkyros): take this out when we decide actual UX
+	if pool.Name == "layered" {
+		if pool.Labels == nil {
+			pool.Labels = map[string]string{}
+		}
+		pool.Labels[ctrlcommon.ExperimentalLayeringPoolLabel] = ""
+	}
+
+	// If this is is a layered pool check to see if we have an image stream
+	// If we do, snarf out the latest image and annotate the pool with it
+	if ctrlcommon.IsLayeredPool(pool) {
+
+		// TODO(jkyros): This will probably requeue the pool because config changed, we should probably do this elsewhere
+		ctrl.experimentalAddEntitlements(pool)
+
+		// Do we have an image stream for this pool? We should if we got here.
+		is, err := ctrl.imageclient.ImageV1().ImageStreams(ctrlcommon.MCONamespace).Get(context.TODO(), pool.Name+ctrlcommon.ImageStreamSuffixMcoContent, metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			// TODO(jkyros): As cgwalters points out, this should probably degrade because it should exist
+			glog.Warningf("Image stream for %s does not exist (yet?): %s", pool.Name, err)
+		} else {
+			// If there is an image ready, annotate the pool with it so node controller can use it if it's the right one
+			err := ctrl.annotatePoolWithNewestImage(is, pool)
```

Comment:
For both this function `annotatePoolWithNewestImage` and `experimentalAddEntitlements` it would have helped me if they were named with something more like ensure or sync in their name so it's clear they're a no-op if everything's good already